jenkinsfiles/ios_jenkinsfile.groovy

```diff
@@ -1,12 +1,17 @@
 /**
 * IOS Jenkinsfile
 */
-// sample values commented below are for https://github.com/feedhenry-templates/helloworld-ios-swift
 
+//     in RHMAP's case, following parameter is sent by RHMAP to Jenkins job.
+//     this means, Jenkins job has to be a parametrized build with that parameter.
+CODE_SIGN_PROFILE_ID = params?.BUILD_CREDENTIAL_ID.trim()                 // e.g. "redhat-dist-dp"
```

Comment:
@odra I am confused now. How does the codesign plugin know which Jenkins credential to use then?